airflow/providers/weaviate/operators/weaviate.py

```diff
@@ -51,21 +54,25 @@ def __init__(
         self,
         conn_id: str,
         class_name: str,
-        input_json: list[dict[str, Any]],
+        input_data: list[dict[str, Any]] | pd.DataFrame,
+        vector_col: str = "Vector",
```

Comment:
We shouldn't edit this operator in a separate PR

Reply:
Sure, I'll do a separate PR for this.

Reply:
Added a separate PR here - https://github.com/apache/airflow/pull/36085